resources/views/profile/view.blade.php

```diff
@@ -0,0 +1,46 @@
+@extends('layouts.main_layout')
+
+@section('head')
+    <title>{{$username}}s Profile</title>
+@endsection
+
+@section('mainBody')
+    @include('inserts.breadcrumb')
+    <div class="container-fluid well">
+        <legend>Profile</legend>
+        <div class="row">
+		    <div class="col-md-2">
+		        <img class="center-block" scr="images/projects/DefaultProfile.jpg">
+		    	<div class="">
+					<h2 class="center"> {{$username}}</h2>
+					<div class="btn-group btn-group-justified">
+			        	<a href="#" class="btn btn-default">Send Message</a>
+			        	<a href="#" class="btn btn-default">Add To Contact</a>
+			    	</div>
+				</div>
+			</div>
+			<div class="col-md-10">
+
+		    	<div class="col-md-6">
+		        	<h4>Basic Information</h4>
+		            <label>User name:</label><p>{{$username}}</p><br/>
+		            <label>Birth Date:</label><p>{{$username }}</p><br/>
```

Comment:
If you set up a eloquent relationship between user and user_profile. You can write something like this to get the data:
`html
<label>Birth Date:</label><p>{{ $user->profile->birthdate }}</p><br/>
`

This would apply to all the data in user_profile